packages/react/src/components/Button/SplitButton/SplitButton.styles.ts

```diff
@@ -27,92 +27,128 @@ export const getStyles = memoizeFunction((theme: ITheme, customStyles?: IButtonS
       getFocusStyle(theme, { highContrastStyle: buttonHighContrastFocus, inset: 2, pointerEvents: 'none' }),
       {
         display: 'inline-flex',
-        selectors: {
-          '.ms-Button--default': {
-            borderTopRightRadius: '0',
-            borderBottomRightRadius: '0',
-            borderRight: 'none',
-            flexGrow: '1',
-          },
-          '.ms-Button--primary': {
-            borderTopRightRadius: '0',
-            borderBottomRightRadius: '0',
-            border: 'none',
-            flexGrow: '1',
+        '.ms-Button--default': {
+          borderTopRightRadius: '0',
+          borderBottomRightRadius: '0',
+          borderRight: 'none',
+          flexGrow: '1',
+        },
+        '.ms-Button--primary': {
+          borderTopRightRadius: '0',
+          borderBottomRightRadius: '0',
+          border: 'none',
+          flexGrow: '1',
 
-            selectors: {
-              [HighContrastSelector]: {
-                color: 'WindowText',
-                backgroundColor: 'Window',
-                border: '1px solid WindowText',
-                borderRightWidth: '0',
-                ...getHighContrastNoAdjustStyle(),
-              },
-              ':hover': {
-                border: 'none',
-              },
-              ':active': {
-                border: 'none',
+          [HighContrastSelector]: {
+            color: 'WindowText',
+            backgroundColor: 'Window',
+            border: '1px solid WindowText',
+            borderRightWidth: '0',
+            ...getHighContrastNoAdjustStyle(),
+          },
+          ':hover': {
+            backgroundColor: 'Highlight',
+            borderColor: 'Highlight',
+            borderRightWidth: '0',
+            color: 'Window',
```

Comment:
```suggestion
            color: 'HighlightText',
```
since the color/bg tokens should be paired together to ensure contrast. `Window` over `Highlight` could work in some themes, and not in others.

Also, I'm probably missing something, but do the `:hover` and `:active` styles need to be nested under the `[HighContrastSelector]` block?